docs/src/main/asciidoc/bigquery.adoc

```diff
@@ -0,0 +1,91 @@
+== Google Cloud BigQuery
+
+https://cloud.google.com/bigquery[Google Cloud BigQuery] is a fully managed, petabyte scale, low cost analytics data warehouse.
```

Comment:
What about also documenting the starter and that it can be used to easily get an auto-wirable `BigQuery` object?
Also, maybe document the `spring.cloud.gcp.bigquery.*` properties like we do for other starters.